tests/data/reference_graphs/quantized/hawq/inception_v3_device_VPU__mode_liberal__setup_type_propagation_based.dot

```diff
@@ -1,527 +1,532 @@
+"54 Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch3x3dbl_3]/NNCFConv2d[conv]/ModuleDict[pre_ops]/UpdateWeight[0]/SymmetricQuantizer[op]/symmetric_quantize" [color=green, id=54, label="8_bit__WFQ_#54", scope="Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch3x3dbl_3]/NNCFConv2d[conv]/ModuleDict[pre_ops]/UpdateWeight[0]/SymmetricQuantizer[op]", style=filled, type=symmetric_quantize];
+"55 Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch3x3dbl_3]/NNCFConv2d[conv]/conv2d" [color=lightblue, id=55, label="conv2d_#55", scope="Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch3x3dbl_3]/NNCFConv2d[conv]", style=filled, type=conv2d];
+"56 Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch3x3dbl_3]/BatchNorm2d[bn]/batch_norm" [id=56, label="batch_norm_#56", scope="Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch3x3dbl_3]/BatchNorm2d[bn]", style=filled, type=batch_norm];
+"57 Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch3x3dbl_3]/RELU" [id=57, label="RELU_#57", scope="Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch3x3dbl_3]", style=filled, type=RELU];
+"58 Inception3/InceptionA[Mixed_5b]/SymmetricQuantizer/symmetric_quantize" [color=green, id=58, label="8_bit__AFQ_#58", scope="Inception3/InceptionA[Mixed_5b]/SymmetricQuantizer", style=filled, type=symmetric_quantize];
+"59 Inception3/InceptionA[Mixed_5b]/avg_pool2d" [id=59, label="avg_pool2d_#59", scope="Inception3/InceptionA[Mixed_5b]", style=filled, type=avg_pool2d];
+"60 Inception3/InceptionA[Mixed_5b]/SymmetricQuantizer/symmetric_quantize" [color=green, id=60, label="8_bit__AFQ_#60", scope="Inception3/InceptionA[Mixed_5b]/SymmetricQuantizer", style=filled, type=symmetric_quantize];
+"61 Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch_pool]/NNCFConv2d[conv]/ModuleDict[pre_ops]/UpdateWeight[0]/SymmetricQuantizer[op]/symmetric_quantize" [color=green, id=61, label="8_bit__WFQ_#61", scope="Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch_pool]/NNCFConv2d[conv]/ModuleDict[pre_ops]/UpdateWeight[0]/SymmetricQuantizer[op]", style=filled, type=symmetric_quantize];
+"62 Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch_pool]/NNCFConv2d[conv]/conv2d" [color=lightblue, id=62, label="conv2d_#62", scope="Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch_pool]/NNCFConv2d[conv]", style=filled, type=conv2d];
+"63 Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch_pool]/BatchNorm2d[bn]/batch_norm" [id=63, label="batch_norm_#63", scope="Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch_pool]/BatchNorm2d[bn]", style=filled, type=batch_norm];
+"64 Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch_pool]/RELU" [id=64, label="RELU_#64", scope="Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch_pool]", style=filled, type=RELU];
+"65 Inception3/InceptionA[Mixed_5b]/cat" [id=65, label="cat_#65", scope="Inception3/InceptionA[Mixed_5b]", style=filled, type=cat];
```

Comment:
This graph looks strange now 
![image](https://user-images.githubusercontent.com/4014476/100849185-422cc800-3493-11eb-8635-565c67969b0f.png)

1) input for AvgPool can't be INT4 by HW config.
2) previously, we had the same quantization parameters on all inputs and outputs of the concat node, which is HW-friendly. now, we allow uncoupled bitwidth of activations on the branches.

how it was before
![image](https://user-images.githubusercontent.com/4014476/100849482-b2d3e480-3493-11eb-9cc3-d0c2073e8cde.png)

Reply:
After enabling heterogenous quantizer configuration handling for HAWQ, I managed to restore most of the reference bitwidth distribution graphs to the same state that is currently present on develop.
Current state of the same piece of the graph that we considered in this discussion:
![Screenshot from 2021-01-07 19-53-58](https://user-images.githubusercontent.com/31695470/103921008-04226400-5123-11eb-8786-9bb42f743f4c.png)
Note that 4-bit AFQ requantizations appeared on the branches. This is due to the fact top-level FQ is 8-bit symmetric per-channel, which is the only configuration listed as supported by avg_pool2d in `vpu.json`, while the conv2d inputs only support 4-bit symmetric per-tensor and 8-bit asymmetric per-tensor quantizations; requantization is unavoidable for current VPU config.